airflow/providers/core/__init__.py

```diff
@@ -0,0 +1,16 @@
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
I am not a fan of "core.sql" too like @ephraimbuddy -- maybe "base.sql" is better 🤷 but I do know that there already is a BaseSQLOperator -- which we should think of moving to a provider too tbh.

But naming is heard. tagging a few people if they have better suggestions for names: @dstandish @jedcunningham @uranusjr

Reply:
Maybe "common" would be better than "core"?

Reply:
We can still change it - before we release the provider and I think we should only do it after we move DBApiHook there (as explained in #23971). 

I think common might be a better choice indeed.